Magento_Theme/templates/html/social-list.phtml

```diff
@@ -0,0 +1,42 @@
+<ul class="list list--horizontal list--center">
+    <li class="list__item">
+        <a class="button button--link button--icon"
+           title="<?php /* @escapeNotVerified */echo __('Facebook') ?>"
+           href="#"
```

Comment:
Different topic, It's not our client project repository, so this task is not relevant.
If you need this module, just add them here - https://github.com/SnowdogApps/magento2-menu